src/AppServiceFileSystem.ts

```diff
@@ -32,17 +36,6 @@ export class AppServiceFileSystem extends AzExtTreeFileSystem<FileTreeItem> {
     }
 
     public async writeFileImpl(_context: IActionContext, node: FileTreeItem, content: Uint8Array, _originalUri: Uri): Promise<void> {
-        const showSavePromptKey: string = 'showSavePrompt';
-        if (getWorkspaceSetting<boolean>(showSavePromptKey)) {
-            const message: string = localize('saveConfirmation', 'Saving "{0}" will update the file "{0}" in "{1}".', node.label, node.client.fullName);
-            const result: MessageItem | undefined = await ext.ui.showWarningMessage(message, DialogResponses.upload, DialogResponses.alwaysUpload, DialogResponses.dontUpload);
-            if (result === DialogResponses.alwaysUpload) {
-                await updateGlobalSetting(showSavePromptKey, false);
-            } else if (result === DialogResponses.dontUpload) {
-                throw new UserCancelledError();
-            }
-        }
-
         let etag: string = nonNullValue(this._etags.get(node.fullId), 'etag');
         try {
             await putFile(node.client, content.toString(), node.path, etag);
```

Comment:
I believe the issue could be solved with just two lines of code...by adding the logs around this call to `putFile`.

I don't think all users of `AzExtTreeFileSystem` want logs, and if they do they might want a different message. We could work around those considerations if we really cared about sharing code, but IMO the "logs" feature doesn't feel important enough to justify the added complexity

Reply:
Ok, yeah I debated if I should just do that. Should have asked if we wanted to share the logs. 👍